### PR TITLE
test addtion for `Netwon Raphson` and `tcycle`

### PR DESCRIPTION
**Pull request by Lakshmikanth2001**

### CODE REVIEW

File: test/test_electricpy.py [hunk 1]
```diff
@@ -411,6 +411,52 @@ def test_syncspeed():
...
+        [10j,-10j]]
+    Vlist = [[1,0],[None,None]]
+    Plist = [None,-2.0]
+    Qlist = [None,-1.0] # 1pu vars consumed
+    F = sim.nr_pq(ybustest,Vlist,Plist,Qlist)
+    X0 = [0,1] # Define Initial Conditions
+    J = sim.jacobian(F) # Find Jacobian
+    # Now use Newton-Raphson to Solve
+    results, iter = sim.NewtonRaphson(F,J,X0)
+    assert_array_almost_equal(results, [-0.236,0.8554], decimal = 3)
```
Comment (engineerjoe440) [inline test/test_electricpy.py:457]: Probably worth a comment to denote where these values (`-0.236`, and `0.8554`) come from. I assume they're from empirical analysis (guess-then-check), but if that's the case, we should just call out something like: `# Values -0.236 and 0.8554 evaluated from empirical analysis`

**Review by engineerjoe440 — CHANGES_REQUESTED** [review]
Looking good! Just want to document how a few "magic" numbers are found.

File: test/test_electricpy.py (hunk 1 above)
Comment (Lakshmikanth2001) [reply test/test_electricpy.py:457]: i frankly didn't get what this function exactly does so i copied the same example from the `doc string` and made it into a test function

Comment (engineerjoe440) [reply test/test_electricpy.py:457]: In fairness... It's been too long for me to remember the specifics too... I'll call it even 😉 haha!

**Review by engineerjoe440 — APPROVED** [review]

**Pull request merged**